To Do List/toDoList.js

```diff
@@ -0,0 +1,22 @@
+const createNewMessage = document.querySelector("#push");
+const newText = document.querySelector("#new__task input");
+const tasks = document.querySelector("#tasks ");
+
+createNewMessage.addEventListener("click", function () {
+  if (newText.value) {
```

Comment:
It's better to go in the opposite direction, like
if(!value) return "error" 
 ....
 your code here